frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1351,47 +1431,32 @@ const AlertsPage_: React.FC<Alerts> = ({ data, loaded, loadError }) => {
       nameFilterID="resource-list-text"
       reduxID="monitoringAlerts"
       Row={AlertTableRow}
-      rowFilters={alertsRowFilters}
+      rowFilters={alertsRowFilters()}
     />
   );
 };
 const AlertsPage = withFallback(connect(alertsToProps)(AlertsPage_));
 
-const ruleHasAlertState = (rule: Rule, state: AlertStates): boolean =>
+export const ruleHasAlertState = (rule: Rule, state: AlertStates): boolean =>
   state === AlertStates.NotFiring ? _.isEmpty(rule.alerts) : _.some(rule.alerts, { state });
 
-const ruleAlertStateFilter = (filter, rule: Rule) =>
+export const ruleAlertStateFilter = (filter, rule: Rule) =>
```

Comment:
It looks like `ruleHasAlertState` and `ruleAlertStateFilter` don't need to be exported now.